components/logo/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
I created a very simple logo component as the old one was not reusable for the design we have.

Reply:
The old logo component is still used on the about page. We might want to get rid of it.

Reply:
Thanks! Will use it when if I can